scripts/rosnav_node.py

```diff
@@ -109,10 +114,28 @@ def __init__(self, ns: Namespace = None):
             "subgoal_mode": self._hyperparams["rl_agent"].get("subgoal_mode", False),
             "ns_to_semantic_topic": rospy.get_param("/train_mode", False),
         }
+        
```

Comment:
might be cleaner to write a helper function to determine the obs_structur based on the agents observation_space as this looks a bit messy

Reply:
Done. @tuananhroman, looks better? Now using the obs_structure for all instantiations of the ObservationManager, also for FlatlandEnv. Tested it in training and agent testing mode. Worked without problems.